docs/contributing.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+Contributions are welcome. We have a few asks of any new contributors.
+
+* Ensure that all changes which affect logic are covered by unit tests. 
+* Do not introduce any unit test failures/regressions.
+* Follow styling as designated in our [styling documentation](/docs/style.md).
+
+Otherwise, we tend to judge code on a case to case basis.
+
+For adding a new game to Archipelago and other documentation on how Archipelago functions, please see 
+[the docs folder](docs/) for the relevant information and feel free to ask any questions in the #archipelago-dev 
```

Comment:
```suggestion
[the docs folder](docs/) for relevant information. Feel free to ask any questions in the #archipelago-dev 
```

Reply:
Not merging that one. I don't like dropping articles if I can help it.